src/site/content/en/fast/image-cdns/index.md

```diff
@@ -101,3 +88,14 @@ Third-party image CDNs provide image CDNs as a service. Just as cloud providers
 There are many good options for image CDNs. Some will have more features than others, but all of them will probably help you save bytes on your images and therefore load your pages faster. Besides feature sets, other factors to consider when choosing an image CDN are cost, support, documentation, and ease of setup or migration.
 
 Trying them out yourself before making a decision can also be helpful. Below you can find codelabs with instructions on how to quickly get started with several image CDNs.
+
+## Effects on Largest Contentful Paint (LCP)
+
+Images are a vital part of the user experience on many websites, and thus factor into how well sites do when it comes to [Largest Contentful Paint](/lcp/). Here are a few things to keep in mind if you decide to use an image CDN:
+
+1. Images served from CDNs may come from a cross-origin server, which involves extra connection setup time. When possible, try to use an image CDN that proxies through the primary origin so that you're not adding extra origins for the browser to connect to. This has the same effect as self-hosting images on the primary origin.
+2. Consider using a the [`fetchpriority` attribute value of `"high"`](/priority-hints/#summary) on the LCP image element so that the browser can begin loading that image as soon as possible.
```

Comment:
It reads a little disjointed now since you accepted my last suggestions, as some of these are general LCP image suggestions. Suggested an extra piece below - WDYT? Too much?

Plus fixed a typo.

```suggestion
2. Consider using the [`fetchpriority` attribute value of `"high"`](/priority-hints/#summary) on the LCP image element so that the browser can begin loading that image as soon as possible. This can benefit images that are served through the primary domain, but also those fetched directly from the image CDN where we want to reduce any extra request time as much as possible.
```